mne/io/cnt/cnt.py

```diff
@@ -145,9 +145,7 @@ def _update_bad_span_onset(accept_reject, onset, duration, description):
             event_type=type(my_events[0]),
             data_format=data_format,
         )
-        duration = np.array(
-            [getattr(e, "Latency", 0.0) for e in my_events], dtype=float
-        )
+        duration = np.zeros_like(my_events, dtype=float)
```

Comment:
I just figured, according to our docs, we can simply pass a float here and all annotations will have the same duration. Hence this should workL

```suggestion
        duration = 0.
```

Reply:
But I'm not sure if `_update_bad_span_onset()` can handle this

Reply:
Yeah, I made that. Should I change to accept a float or just leave it how it is?